apps/system/lockscreen/js/lockscreen_notification_builder.js

```diff
@@ -59,97 +59,19 @@
     }
 
     node.addEventListener('click',
-      this.requestHighlight.bind(this, node));
+      this.requestActivate.bind(this, node));
```

Comment:
this code seems never will be called, since there won't be any ``click`` events in non-touch environment.